_shared/hooks/post_gen_project.py

```diff
@@ -144,12 +169,13 @@ def main():
         template_ignore_patterns.extend([
             "{{ cookiecutter.package_name }}/__init__.py",
             "{{ cookiecutter.package_name }}/app.py",
+            "{{ cookiecutter.package_name }}/models/__init__.py",
             "Dockerfile",
             "package.json",
             "yarn.lock",
             ".docker.env",
-            "{{ cookiecutter.package_name }}/pshell.py",
             "tests/__init__.py",
+            "tests/factories/__init__.py",
```

Comment:
Same as with `models/__init__.py`, the cookiecutter needs to generate an empty `factories/__init__.py` in new projects in order to get the tests to pass but we want projects to edit this file directly so we don't want `make template` to update it.